packages/MSLG/README.MD

```diff
@@ -0,0 +1,107 @@
+# mslg
+## Please notice that this package is still in Preview.
+mslg tool has the ability to parse a .LG file that adheres to the LG file format. CLI tool should report an error on an statically detectable invalid .LG file. MSLG tool should also have the ability to generate a collated .LG file by parsing all provided (or found) .LG files. MSLG tool can also list all possible template expansions for a given template and test data for entity references.
```

Comment:
Consider re-writing this to MSLG CLI tool: 
- Reports an error on invalid .lg file. All errors that are statically detectable are reported.
- Can generate a collated .lg file by parsing and analyzing more than one .lg file
- Helps expand any given template (or all templates in a file) based on test values for entity references.

Reply:
fixed